packages/client-utils/src/chain.js

```diff
@@ -37,3 +41,56 @@ export const pollBlocks = opts => async lookup => {
     }
   }
 };
+
+/**
+ * @import {CosmosAPI} from './grpc-rest-api.js';
+ * @import {IntervalIO} from './clock-timer.js';
+ */
+/**
+ * @param {CosmosAPI} api
+ * @param {number} [height]
+ */
+export const queryBlock = (api, height) =>
+  api.getJSON(`/cosmos/base/tendermint/v1beta1/blocks/${height || 'latest'}`);
+
+/**
+ * @param {CosmosAPI} api
+ * @param {number} [delta]
+ */
+const recentBlockRate = async (api, delta = 2) => {
+  const relevant = ({ height, time }) => ({ height, time });
```

Comment:
interesting pattern for `_.pick`. No concerns here but we might want to sync before adopting it more widely (or in order to accelerate that if it's a good idea)